Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Base image for container
```

Comment:
Adicionalmente, es conveniente que sepáis realmente qué estáis usando. [Esta es la plantilla para las imágenes oficiales](https://github.com/docker-library/golang/blob/master/Dockerfile-linux.template) ¿Necesitas todo lo que hay ahí?  Por otro lado, ¿has visto si hay instalado un gestor de tareas que puedas usar? ¿O si hay un usuario ya creado?

Reply:
Viendo las layers [aquí](https://hub.docker.com/layers/library/golang/alpine/images/sha256-3325c5593767d8f1fd580e224707ca5e847a1679470a027aaa3c71711ce16613?context=explore), la 0 y la 2 instalan paquetes de alpine, entre ellos `apk`, el gestor de paquetes por defecto de la distribución y en la 7 se instala go, con el test runner que uso `go test` y el gestor de dependencias `go mod`. En cuanto a creación de usuarios, no parece

Reply:
Puedes ver las layers ahí como dices, que están bastante incompletas, o puedes mirar el Dockerfile que te he enlazado y ver que, por ejemplo, se instala make. 

https://github.com/docker-library/golang/blob/dae3405a325073e8ad7c8c378ebdf2540d8565c4/Dockerfile-linux.template#L153

Aparte de los compiladores de c y c++. ¿Necesitas eso para tus tests?

Por otra parte ¿realmente crees que rebatir los comentarios que hago en vez de actuar o tenerlos en cuenta contribuye a que yo entienda si has superado o no los diferentes objetivos de aprendizaje aquí?

Reply:
Resuelto aquí [3be7b2f](https://github.com/diagmatrix/maybe-better-maybe-worse/pull/29/commits/3be7b2f4d4289b103295648749adb7b8c9005824)

En cuanto a rebatir, no era mi objetivo con el comentario, aunque re-leyendo es un poco pasivo-agresivo. Perdona